ci/travis.sh

```diff
@@ -119,3 +126,6 @@ build_and_test
 
 CHANNEL=1.13.0
 build_only
+
+CHANNEL=stable
+build_core_test
```

Comment:
nit: I think I'd prefer `build_core_test` to just go inline between lines 125 and 126 instead of reactivating stable